src/main/java/org/web3j/model/Greeter.java

```diff
@@ -0,0 +1,179 @@
+package org.web3j.model;
```

Comment:
Should this be in the project if we're using the web3j Maven plugin? I don't believe it is if we use the Gradle plugin

Reply:
Removed generated classes as the maven plugin will generate them upon request